html/dist/voiceChat.css

```diff
@@ -1,17 +1,12 @@
-﻿
-* {
+﻿* {
     margin: 0 0;
     padding: 0 0;
     border-spacing: 0px;
     box-sizing: content-box;
 }
-@font-face {
-    font-family: "GTAV_font_6";
-    src: url('./font6.ttf') format("truetype");
-}

```

Comment:
You told me on discord that the font was not there, but actually it IS int he dist folder

Reply:
Fixed.